packages/backend/src/database/migrations/20240424131220_populate_slugs.ts

```diff
@@ -0,0 +1,45 @@
+import { Knex } from 'knex';
```

Comment:
Running `populate` data in a separate migration, because I remember last time we did something similar, we had problems with the migration because the trasnaction was not committed